indexer/services/roundtable/src/tasks/subaccount-username-generator.ts

```diff
@@ -0,0 +1,34 @@
+import { logger } from '@dydxprotocol-indexer/base';
+import {
+  SubaccountUsernamesTable,
+  SubaccountsWithoutUsernamesResult,
+} from '@dydxprotocol-indexer/postgres';
+import { generateUsername } from 'unique-username-generator';
+
+const RANDOM_DIGITS = 3;
+
+export default async function runTask(): Promise<void> {
+  const subaccounts:
+  SubaccountsWithoutUsernamesResult[] = await
+  SubaccountUsernamesTable.getSubaccountsWithoutUsernames();
+  for (const subaccount of subaccounts) {
+    const username: string = generateUsername('', RANDOM_DIGITS);
```

Comment:
There are ~16k subaccounts in dydx chain today, would that cause an issue for this?

Reply:
I calculated current username space as 50 Billion accounts. Should be fine if we are in the 1000s